src/components/Navigator/NavigatorCard.vue

```diff
@@ -1013,26 +1018,31 @@ $navigator-card-vertical-spacing: 8px !default;
   }
 }
 
-.technology-title {
+:deep(.navigator-title) {
```

Comment:
Does this really need to be a deep rule? Just checking.

Reply:
Yes, to make sure the parent component also can use it.